src/main/java/edu/wright/airviewer2/AIRViewerController.java

```diff
@@ -342,5 +354,65 @@ public void initialize(URL url, ResourceBundle rb) {
         });
 
     }
+    
+    @FXML private TextField textFieldValue;
+    
+    @SuppressWarnings("deprecation")
+   @FXML private void download() throws IOException {
+    System.out.println("textfieldvalue"+textFieldValue.getText());
+    Splitter splitter = new Splitter();
+   // FileChooser fileChooser = new FileChooser();
+   // FileChooser.ExtensionFilter extFilter = new FileChooser.ExtensionFilter("PDF files (*.pdf)", "*.pdf");
+   // fileChooser.getExtensionFilters().add(extFilter);
+   // 
+   // File file = fileChooser.showSaveDialog((Stage) pagination.getScene().getWindow());
+    DirectoryChooser dirChooser = new DirectoryChooser();
+    dirChooser.setTitle("Select a folder");
+    File selectedDir = dirChooser.showDialog((Stage) pagination.getScene().getWindow());
+    String selectedDirPath =selectedDir.getAbsolutePath();
+    SimpleDateFormat sf = new SimpleDateFormat("ddmmyyyHHMMSS");
+    
+    PDFMergerUtility PDFmerger = new PDFMergerUtility();
+    PDDocument document = PDDocument.load(new File(model.getPathName()));
+    List<PDDocument> pages = splitter.split(document);
+    Iterator<PDDocument> iterator = pages.listIterator();
+    //Saving each page as an individual document
+    
+    
+    PDDocument document1 = new PDDocument();
+    OutputStream out = new ByteArrayOutputStream();
+    boolean flag = false;
+    String[] values = textFieldValue.getText().split(",");
+    System.out.println("valueslength"+values.length);
+    int count =1;
+    for(int i=0;i<values.length;i++) {
+    System.out.println("i..."+values[i]);
+    for(int j=0;j<pages.size();j++) {
+    System.out.println("j..."+j);
+    if(values[i].equals((j+1)+"")) {
+    System.out.println("values.........."+values[i]);
+    
+    File tempfile = new File(selectedDirPath + "/" + "temp_"+values[i]+".pdf");
+    flag = true; 
+    PDDocument pd = pages.get(j);
+    pd.save(tempfile);
+    PDFmerger.addSource(tempfile);
+    
+    
+    }
```

Comment:
It would be great if you format the code before committing the file.

Reply:
Formatted the code.